packages/flutter_tools/templates/app/linux.tmpl/app.desktop.tmpl

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name={{projectName}}
+Comment={{description}}
+Icon=icon.png
```

Comment:
It's renamed in the install step, so this won't be right. Also, doesn't this need to be the full path to the icon, added by a script during the install step?